nrpy/helpers/colored.py

```diff
@@ -0,0 +1,65 @@
+"""
+Placeholder.
+
+Author: Steven Brandt
+"""
+
+import sys
+from typing import Any, Dict
+from typing_extensions import Literal
+
+color_names = Literal["red", "green", "yellow", "blue", "magenta", "cyan"]
+
+
+def not_colored(arg: Any, c: color_names) -> str:  # pylint: disable=unused-argument
```

Comment:
remove_colored()?

Reply:
To me "remove" would mean that the method is going away. Since it's not, that doesn't make sense to me. I could go with disable_colored() if you prefer. I don't think it's any less/more clear, though.

Reply:
how about remove_coloring()? I always want to avoid using negations as therein lies the path to double negatives.

Reply:
this would be a nice counter to my proposed renaming of colored -> add_coloring

Reply:
alternatively it could be with_coloring / without_coloring

Reply:
The functionality is alternative to the termcolor package which does not disable coloring when I think it should. I'd prefer to keep their name for "colored()" for that reason. So I'd like to go with colored/remove_colored, although IMHO not_colored is just as clear in meaning.